src/lib/utils.ts

```diff
@@ -37,12 +38,15 @@ const extractErrorText = (error: any) => {
   return JSON.stringify(error);
 };
 
-export const collectError = (description: string, error?: any) => {
+export const collectError = (description: string, error?: any, additionalInformation?: { [name: string]: string }) => {
   if (browser) {
-    console.error(`Error: ${description}`);
     gtag("event", "exception", {
       description,
       errorText: error && extractErrorText(error),
     });
+    fetch("/api/monitoring/error", {
+      method: "POST",
+      body: JSON.stringify({ description, error: extractErrorText(error), additionalInformation }),
+    });
```

Comment:
I added the option to add more information when we collect an error, and it now also will post that error to the new endpoint that just logs it on the server